assets/kube-apiserver/kube-apiserver-configmap.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
Does this file need to be added to the list of files rendered [here](https://github.com/openshift/ibm-roks-toolkit/blob/master/pkg/render/manifests.go#L88)?

Reply:
I wil investigate this. Thanks!

Reply:
Okay, I understand now: this file is not used at all that's why it is not rendered.

Reply:
I have re-organized this a bit. The egress config has its own config map now. Thanks!